docs/store/index.html

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Store
+permalink: /store
+order: 40
+---
+
+<!-- HEADER -->
+{% include header.html with_signup_form='true' has_sticky_navigation='true' background_color='#7F7E9E' %}
+
+<style>{% include css/store.css %}</style>
+
+<div class="hero">
+  <h1 class="heading">Stack and Stencil Hub</h1>
+  <h2 class="subheading">Start building your stacks easily with these hundereds of examples</h2>
+  <div class="search-container">
+    <div class="dropdown">
+      <a href="#" class="dropdown-selection">
+        <label>In Stencils</label><span class="dropdown-icon"></span>
+      </a>
+      <div class="dropdown-options">
+        <a href="#">In Stacks</a>
+        <a href="#">In Stencils</a>
+      </div>
+    </div>
+    <div class="search-bar">
+      <div class="search-bar-field">
+        <input placeholder="Search a keyword or name..." /><a href="#" class="search-icon"></a>
+        <a href="javascript:;" class="remove-icon"></a>
+      </div>
+      <div class="search-results">
+        <div class="search-results-rows">
+        </div>
+        <a href="#" class="see-all-results-row">
+          See all results...
+        </a>
```

Comment:
you can keep headers, anchors in a single line `<a href="#" class="see-all-results-row">See all results...</a>` to avoid weird whitespace issues.